src/modules/Providers/views/details/tabs/VirtualMachines/utils/filters/VsphereHostFilter.ts

```diff
@@ -7,7 +9,7 @@ const labelToFilterItem = (label: string): EnumValue =>
  * This component enables filtering the VMware's virtual machines
  * by the hostname that they are running on.
  */
-export const VsphereHostFilter = (t: (string) => string) => {
+export const vsphereHostFilter = (t: TFunction) => {
```

Comment:
I think it is easier instead of adding a type, simply import t directly here and omit it from args